### PR TITLE
Module 13

### PR DESCRIPTION
**Pull request by A-J1993**

### CODE REVIEW

File: main.tf
Comment (JackMeadDev) [inline main.tf:57]: I can't see any evidence that this is actually used anywhere!

File: .travis.yml
Comment (JackMeadDev) [inline .travis.yml]: Nice, I can see that Terraform is now deploying successfully as part of your pipeline 😄

File: todo_app/app.py
Comment (JackMeadDev) [inline todo_app/app.py:46]: I don't think you ever use this logger, so no harm but we might as well remove this line

File: todo_app/app.py
Comment (JackMeadDev) [inline todo_app/app.py:48]: I think you've skipped Step 3: setting the log level - happy for you to apply this before merging since the code is given but we should use this since we've configured the environment variable in Terraform! I've assumed in the snippet below that you'll read the environment variable directly, and set a default of INFO to match your Terraform default. Try switching this variable and check that it only logs the relevant messages
```
app.logger.setLevel(os.getenv('LOG_LEVEL', 'INFO'))
```

File: todo_app/app.py
Comment (JackMeadDev) [inline todo_app/app.py:94]: I know we discussed on Slack, but this is a nice approach to handling the error - we log the error, and spell out the issue to the user

File: todo_app/app.py
Comment (JackMeadDev) [inline todo_app/app.py]: We might as well get rid of this properly now

**Review by JackMeadDev — APPROVED** [review]
Nice work, I can see your logs were working in Loggly, and I had no issues getting it running for me too. I can also see that you've successfully got Terraform making changes in your pipeline - nice! I've left a few optional thoughts below, and it would be worthwhile ensuring the Logging level is configurable, but otherwise I'm happy for you to merge this when you're ready

File: main.tf
Comment (A-J1993) [reply main.tf:57]: Adding `LOG_LEVEL` variable to the `travis.yml` file

File: todo_app/app.py
Comment (A-J1993) [reply todo_app/app.py:48]: Added to line 47

File: main.tf
Comment (A-J1993) [reply main.tf:57]: With default of `INFO`

File: todo_app/app.py
Comment (A-J1993) [reply todo_app/app.py:46]: Use it in some of the cases? like `app.logger.info("Card (" + str(cards.find_one({"_id":ObjectId(_id)}).get("name")) + ") Completed")`

**Pull request merged**
